netlify.toml

```diff
@@ -4,6 +4,8 @@ command = "yarn build"
 
 [context.production.environment]
 VUE_APP_MULTINET_HOST = "https://api.multinet.app/api"
+VUE_APP_MULTINET_CLIENT = "https://multinet.app/"
 
 [context.deploy-preview.environment]
 VUE_APP_MULTINET_HOST = "https://api.multinet.app/api"
+VUE_APP_MULTINET_CLIENT = "https://multinet.app/"
```

Comment:
I think it's ok to have it split up like this, since it makes it more explicit that these *might* vary in the future.

For instance, shouldn't the `HOST` variable be pointing to a testing API?